build.gradle

```diff
@@ -20,21 +13,77 @@ buildscript {
     }
 }
 
-dependencies {
-    compile "org.elasticsearch:elasticsearch:7.5.1"
-    compile "org.elasticsearch.plugin:parent-join-client:7.5.1"
-    compile "org.jetbrains.kotlin:kotlin-stdlib:1.3.61"
-    testCompile "org.apache.logging.log4j:log4j-api:2.13.0"
-    testCompile "org.apache.logging.log4j:log4j-core:2.13.0"
-    testCompile 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
-    testCompile "junit:junit:4.13"
-}
+group 'mbuhot'
+version '1.0.0beta1'
+
+apply plugin: 'java-base'
+apply plugin: 'kotlin'
+//apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
 
 repositories {
     mavenCentral()
+
+    maven {
+        def repo_url = project.hasProperty('repo_url') ? project.property('repo_url')
+                : 'https://nexus.semandex.net:44355/repository/semandex-maven-prod/'
+        url repo_url
+        allowInsecureProtocol = true
```

Comment:
We dont need this anymore since our repo is secured now. Please take it out and check if it works

Reply:
this is what we have in gradle-script-plugin now. It will be replaced soon. let's keep it as-is